libmscore/staff.cpp

```diff
@@ -455,6 +468,25 @@ TimeSig* Staff::nextTimeSig(const Fraction& tick) const
       return (i == timesigs.end()) ? 0 : i->second;
       }
 
+
+//---------------------------------------------------------
+//   currentTimeSigTick
+//
+//    return the tick position of the time sig currently
+//    in effect at tick
+//---------------------------------------------------------
+
+Fraction Staff::currentTimeSigTick(const Fraction& tick) const
+      {
+      if (timesigs.empty())
+            return Fraction(0,1);
```

Comment:
space after comma

Reply:
@JoshuaBonn1 Fraction(0,1) seems to be an idiom and is written like that in many other places. 
74 matches for Fraction(0, 1) as opposed to 399 for Fraction(0,1)

Reply:
In that case, use a space after a comma. The standard coding practice across all the code is much more prevalent than someone who forgot a space. Since there is a chance of making it Fraction::zero, I understand that it will probably be replaced. 

Definitely not merge stopping though.

Reply:
@JoshuaBonn1 Ok. If there are more substantial comments, I will fix this along with them.